src/chrome/content/rules/ComputerWorld.xml

```diff
@@ -1,85 +1,56 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://computerworld.com/ => https://www.computerworld.com/: (51, "SSL: no alternative certificate subject name matches target host name 'www.computerworld.com'")
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://computerworld.com/ => https://www.computerworld.com/: (28, 'Connection timed out after 10001 milliseconds')
 	For other IDG coverage, see IDG.se.xml.
 
-
 	CDN buckets:
-
 		- computerworld.com.edgesuite.net/newsletter_images/
 		- computerworld.com.edgesuite.net/techbrief/
-
 		- redirect.etouches.com
-
 			- events.computerworld.com
-
 		- computerworld.jobamatic.com
-
-			- itjobs.compterworld.com
-
+			- itjobs.computerworld.com
 		- computerworld.mobify.me
-
 			- m.computerworld.com
-
 		- computerworld.idg.netdna-cdn.com
-
-			- -ssl doesn't exist
+			- SSL doesn't exist
 			- cdn[1-4].computerworlduk.com
 
-
 	Nonfunctional domains:
-
 		- blogs.computerworld.com *
-		- resources.computerworld.com	(refused)
+		- resources.computerworld.com (refused)
 		- (www.)computerworlduk.com *
 		- cdn[1-4].computerworlduk.com	(404; mismatched, CN: *.netdna-ssl.com)
```

Comment:
Does not exist anymore

Reply:
@J0WI https://cdn1.computerworlduk.com, https://cdn2.computerworlduk.com, https://cdn3.computerworlduk.com and https://cdn4.computerworlduk.com WFM, so I moved them to the corresponding target tags.